Snakes.py

```diff
@@ -139,16 +160,28 @@ def gameLoop():
         y+=y_change
         gameDisplay.fill((255,255,255))
         pygame.draw.rect(gameDisplay, (0,255,0), [foodX,foodY,10,10])
-        
+
         snakeHead=[]
         snakeHead.append(x)
         snakeHead.append(y)
         snakeList.append(snakeHead)
+
+        for i in block:
+            pygame.draw.rect(gameDisplay, (255, 0, 0), [i[0], i[1], 10, 10])
+            if i == snakeHead:
+                pyOver = True
+                lossreason='Ooops You Hit a BLOCKER'
+                sound = pygame.mixer.Sound("Point.wav")
+                sound.play()
+
         if len(snakeList)>snakeLen:
             del snakeList[0]
         for body in snakeList[:-1]:
             if body==snakeHead:
                 pyOver=True
+                lossreason ='Oooops You Hit YOURSELF'
+                sound = pygame.mixer.Sound("Point.wav")
```

Comment:
Please rebase and then change this path to the f]proper file present in assets folder